lnst/Recipes/ENRT/RDMARecipe.py

```diff
@@ -0,0 +1,79 @@
+from socket import AF_INET
+
+from lnst.Common.IpAddress import interface_addresses
+from lnst.Common.Parameters import IPv4NetworkParam, IntParam, StrParam
+from lnst.Controller import HostReq, DeviceReq, RecipeParam
+from lnst.RecipeCommon.Perf.Measurements import RDMABandwidthMeasurement
+from lnst.RecipeCommon.Perf.Measurements.RDMABandwidthMeasurement import RDMAFlow
+from lnst.RecipeCommon.Perf.Recipe import RecipeConf, Recipe as PerfRecipe
+
+
+class RDMARecipe(PerfRecipe):
+    """
+    RDMA / InfiniBand bandwidth test recipe
+    """
+
+    driver = StrParam(default="mlx5_core")
```

Comment:
Maybe change this to not have default and have `mandatory=True` to force user to specify it.

Reply:
Now inheriting from `SimpleNetworkRecipe`. This is not relevant anymore.